contracts/base/Escrow.sol

```diff
@@ -198,6 +199,47 @@ contract Escrow is EscrowInterface, TInitializable, BaseEscrowDapp {
         emit TokensClaimed(recipient);
     }
 
+    /**
+        claimTokens(address recipient, uint256 value{denoted in collateral token}) 
+        - checks loan is closed 
+        - the loan must be liquidated 
+        - and the sender should be the loans contract instance 
+        
+     */
+    /**
+        @notice Send the equivilant of tokens owned by this escrow (in collateral value) to the recipient,
+        @dev The loan must not be active
+        @dev The loan must be liquidated
+        @dev The recipeient must be the loans contract
+        @param recipient address to send the tokens to
+        @param value The value of escrow held tokens, to be claimed based on collateral value
+      */
+    function claimTokensByCollateralValue(address recipient, uint256 value) external {
+        require(getLoan().status != TellerCommon.LoanStatus.Active, "LOAN_ACTIVE");
```

Comment:
I would change this to require the loan is `Closed`

Reply:
Changed